src/test/java/spoon/test/exceptions/ExceptionTest.java

```diff
@@ -101,4 +107,22 @@ public void testExceptionDuplicateClass() throws Exception {
 					.build();
 	}
 
+	@Test
+	public void testUnionCatchExceptionInsideLambdaInNoClasspath() {
+		// contract: the model should be built when defining a union catch inside a lambda which is not known (noclasspath)
+		// and the catch variable should be the same than outside a lambda
```

Comment:
I added this contract which seems reasonable, but I have problem to achieve this second part of the contract (and so, the last equals is a wrong assertion): JDT does not provide any information about the exception type inside the lambda, even not the fully qualified name of the exception. As a result, I have only the simple name of the class, which does not allow me easily to infer any information like on any other case. 

So, do you have any idea how to improve that, or could we remove this part of the contract and consider that in this specific case the model have less information?